lib/matplotlib/axes/_axes.py

```diff
@@ -3861,6 +3860,11 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
         linewidths : scalar or array_like, optional, default: None
             If None, defaults to (lines.linewidth,).
 
+        verts : sequence of (x, y), optional
+            If `marker` is None, these vertices will be used to
+            construct the marker.  The center of the marker is located
+            at (0,0) and the size is normalized.
```

Comment:
I think I know, but normalized from what to what? Also, "the size" (as an unqualified term) could be confused with the `s` parameter.